__tests__/integration.test.js

```diff
@@ -70,6 +70,14 @@ describe("Integration tests", () => {
 						});
 					});
 			});
+			test("GET:200 article sent to client includes comment_count property", () => {
+				return request(app)
+					.get("/api/articles/3")
+					.expect(200)
+					.then(({ body: { article } }) => {
+						expect(article).toHaveProperty("comment_count", 2);
+					});
+			});
```

Comment:
Great that because of your flexible tests previously, you were able to write a separate test for additional functionality and didn't have to edit previous tests 🙌